demos/form/form.php

```diff
@@ -56,13 +57,16 @@
 $group->addControl('surname');
 $group->addControl('gender', [Form\Control\Dropdown::class, 'values' => ['Female', 'Male']]);
 
+$accordionLayout = $form->layout->addSubLayout([\Atk4\Ui\Form\Layout\Section\Accordion::class]);
```

Comment:
CS fixer is a little short here, we prefer `Form\...` import here, never FQCN, except single FQCN like `\Throwable`. You will get feeling for the CS we use by reading more code.